.travis.yml

```diff
@@ -11,3 +11,4 @@ before_install:
   - sudo apt-get update -qq
```

Comment:
managing to get rid of sudo would be great because that's what makes travis run on their old infrastructure.